.github/workflows/devcontainer.yml

```
@@ -0,0 +1,89 @@
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    env:
+      IMAGE_PATH: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)